parcels/grid.py

```diff
@@ -167,7 +167,7 @@ def add_Sdepth_periodic_halo(self, zonal, meridional, halosize):
                 assert self.depth.shape[2] == self.ydim, "Third dim must be y."
 
     def computeTimeChunk(self, f, time, signdt):
-        nextTime_loc = np.infty * signdt
+        nextTime_loc = np.infty if signdt >= 0 else -np.infty
```

Comment:
0*inf was nan. It was breaking? I thought that the nan, was just ignored later in the min(next_times) of pset.execute()